run_rodent.py

```diff
@@ -10,45 +10,41 @@
 
 from stac_mjx import main
 from stac_mjx import utils
+from pathlib import Path
+
```

Comment:
I should have mentioned in this in the previous PR, but do you mind adding a doc string to explain what the intended use of this file is? It's not mentioned in the README.md, so its existence is a little mysterious.

Reply:
Makes sense, will do, and add a mention in the README. I also forgot to update the existing code blocks in the README with the new examples, so will do that as well.